newdle/notifications.py

```diff
@@ -55,3 +74,28 @@ def create_participant_email(
     )
     msg.attach_alternative(html_content, 'text/html')
     return msg
+
+def create_creator_email(
```

Comment:
This function is almost identical with `create_participant_email` - see if you can't use one function that can cover both cases